RELEASE-NOTES.md

```diff
@@ -1,3 +1,24 @@
+# Xively Client version 1.3.3
+#### Jan 22 2018
+
+## Features
+- [CC3220SF] Updated example application support to TI SimpleLink SDK 1.6, XDC Tools version 3.50.04.43.
+- [CC3220SF] Example application now detects WiFi disconnections and cleanly reconnects when WiFi AP returns.
+- [ESP32] Updated the Xively C Client build to work with IDF SDK: Master commit af63ca1
+- [ESP32] Improved LED signaling in demo application.
+- [Secure File Transfer] CBOR library assertions are now suppressed.  They can be enabled by defining `XI_DEBUG_ASSERT=1` on the make command line when building the Xively C Client library. 
+
+## Bugfix
+- [ESP32] Fix integer overflow in time BSP.
+- [ESP32/CC3200] Fix FWU Board Support Packages for SFT when running an SFT migration that does not include new firmware binaries. Previously the device rebooted upon all SFT deliveries, now it's only rebooted if a new firmware image was downloaded.
```

Comment:
Now that I'm reading this, I may not have been clear enough in my PR's release notes.

I'm not sure about the CC3200, but this bug also switched OTA partitions in the ESP32. Updating a config file would get the devices to run an older firmware, so it's more serious than just a reboot.